test/verify/check-journal

```diff
@@ -293,9 +293,14 @@ ExecStart=/bin/sh -c 'for s in $(seq 10); do echo SLOW; sleep 0.1; done; sleep 1
                         ])
 
         # Check selecting of services
-        if m.image == "rhel-7-6-distropkg": # Introduced in #8347
+        if m.image == "rhel-7-6-distropkg": # Introduced in #8347 and #11675
             return
 
+        b.go("#/?start=previous-boot&tag=check-journal")
+        wait_log_lines([expected_date,
+                        ["check-journal", "BEFORE BOOT", ""]
+                        ])
```

Comment:
Would be nice to add a `not_expected` argument to `wait_log_lines()` to ensure that here e. g. "AFTER BOOT" does *not* occur. But this is also fine for a follow-up.

Reply:
This is checked by `wait_log_lines`. It check that only these lines are there and nothing else. (thats also why the test failed because in the journal was also the button `Show older entries` but I didn't put it into the list of expected log lines)